variables.yml

```diff
@@ -24,16 +30,18 @@ networks:
       min_self_del:
         dance: '10,000'
         planck: 10000000000000000
+    session:
+      blocks: 300
+      hours: 1
     staking:
-      staking_session_delay:
-        sessions:
-          num: 2
-          display: 'two'
-        blocks: 600
-        hours: 
-          num: 2
-          display: 'two'
-    oracles:
+      joining_delay_sessions: 2
+      joining_delay_blocks: 600
+      joining_delay_hours: 1
+      leaving_delay_sessions: 2
+      leaving_delay_blocks: 600
+      leaving_delay_hours: 1
+      max_buffer_size: 100
```

Comment:
this variable doesn't look like it's used?

Reply:
yes it is not used. I'll remove it